applications/RANSApplication/custom_processes/rans_nut_k_omega_sst_update_process.cpp

```diff
@@ -171,7 +169,9 @@ double RansNutKOmegaSSTUpdateProcess::CalculateElementNuT(
     const double beta_star = rProcessInfo[TURBULENCE_RANS_C_MU];
     const double a1 = rProcessInfo[TURBULENCE_RANS_A1];
 
-    const auto& r_geometry = rElement.GetGeometry();
+    auto& r_geometry = rElement.GetGeometry();
+    const auto& r_properties = rElement.GetProperties();
+    auto constitutive_law = rElement.GetValue(CONSTITUTIVE_LAW);
```

Comment:
Why don't retrieving the constitutive law from the properties? I think it's more natural in fluid dynamics.